app/models/challenge.rb

```diff
@@ -1,3 +1,4 @@
 class Challenge < ApplicationRecord
   has_many :groups
+  validates :name, :description, :icon
```

Comment:
shouldn't there be a "presence: true" in the end? 
in my latest schema there is no field :icon?